framework/src/task_manager/task_manager_internal.h

```diff
@@ -58,7 +67,8 @@ struct task_list_s {
 typedef struct task_list_s task_list_t;
 
 struct task_list_data_s {
-	int builtin_idx;
+	int task_type;
```

Comment:
for simplicity, I recommend just 'type' and 'idx' instead of 'task_type' and 'task_idx'.